asreview/webapp/utils/project.py

```diff
@@ -60,13 +57,6 @@ def init_project(
                 'authors': project_authors
             }, fp)
 
-        asr_kwargs = deepcopy(app.config['asr_kwargs'])  # remove config
-        with open(get_kwargs_path(project_id), "w") as fp:
-            json.dump(asr_kwargs, fp)
-
-        # make a copy of the arguments to the state file
-        asr_kwargs['state_file'] = str(get_state_path(project_id))
```

Comment:
@qubixes I'm a bit clueless on this line. Is this still required?

Reply:
It seems not, so it should be safe to delete.

Reply:
Done